packages/frontend/src/App/Footer/index.tsx

```diff
@@ -12,8 +12,8 @@ const Footer = () => (
         target="_blank"
         rel="noopener noreferrer"
       >
-        <img src={twitch} className={styles.twitchLogo} alt="twitch" /> Follow
```

Comment:
I thought there was an agreement to remove promotion of personal stuff.

Reply:
I'll throw it behind a couple ENV vars.